Runtime/Scripts/Internal/FFIClients/FfiRequestExtensions.cs

```diff
@@ -0,0 +1,296 @@
+                    ffiRequest.CreateVideoTrack = createVideoTrackRequest;
+                    break;
+                case CreateAudioTrackRequest createAudioTrackRequest:
+                    ffiRequest.CreateAudioTrack = createAudioTrackRequest;
+                    break;
+                case GetStatsRequest getStatsRequest:
+                    ffiRequest.GetStats = getStatsRequest;
+                    break;
+                // Video
+                case AllocVideoBufferRequest allocVideoBufferRequest:
+                    ffiRequest.AllocVideoBuffer = allocVideoBufferRequest;
+                    break;
+                case NewVideoStreamRequest newVideoStreamRequest:
+                    ffiRequest.NewVideoStream = newVideoStreamRequest;
+                    break;
+                case NewVideoSourceRequest newVideoSourceRequest:
+                    ffiRequest.NewVideoSource = newVideoSourceRequest;
+                    break;
+                case CaptureVideoFrameRequest captureVideoFrameRequest:
+                    ffiRequest.CaptureVideoFrame = captureVideoFrameRequest;
+                    break;
+                case ToI420Request toI420Request:
+                    ffiRequest.ToI420 = toI420Request;
+                    break;
+                case ToArgbRequest toArgbRequest:
+                    ffiRequest.ToArgb = toArgbRequest;
+                    break;
+                // Audio
+                case AllocAudioBufferRequest allocAudioBufferRequest:
+                    ffiRequest.AllocAudioBuffer = allocAudioBufferRequest;
+                    break;
+                case NewAudioStreamRequest wewAudioStreamRequest:
+                    ffiRequest.NewAudioStream = wewAudioStreamRequest;
+                    break;
+                case NewAudioSourceRequest newAudioSourceRequest:
+                    ffiRequest.NewAudioSource = newAudioSourceRequest;
+                    break;
+                case CaptureAudioFrameRequest captureAudioFrameRequest:
+                    ffiRequest.CaptureAudioFrame = captureAudioFrameRequest;
+                    break;
+                case NewAudioResamplerRequest newAudioResamplerRequest:
+                    ffiRequest.NewAudioResampler = newAudioResamplerRequest;
+                    break;
+                case RemixAndResampleRequest remixAndResampleRequest:
+                    ffiRequest.RemixAndResample = remixAndResampleRequest;
+                    break;
+                case E2eeRequest e2EeRequest:
+                    ffiRequest.E2Ee = e2EeRequest;
+                    break;
+                default:
+                    throw new Exception($"Unknown request type: {request?.GetType().FullName ?? "null"}");
+            }
+        }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static void Clear<T>(this FfiRequest ffiRequest, T request)
+        {
+            switch (request)
```

Comment:
I think you could use `ffiRequest.ClearMessage();` instead of the switch. It does effectively the same in a private manner: nullifies polymorphic `message_`